### PR TITLE
feat: view application in open / returned state (FPLA-1990)

### PR DESCRIPTION
**Pull request by Jamesnelson117**
**Before creating a pull request make sure that:**

- [x] PR's title includes JIRA ticket number and follow [Conventional Commits](https://www.conventionalcommits.org/) specification
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-1990

### Change description ###

Adds 'view application' tab
View application tab present in Open and Returned state
Hides all tabs (apart from confidential and overview) in open and returned state

Screenshot 2020-07-21 at 13 22 35

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json [hunk 1]
```diff
@@ -936,5 +936,252 @@
...
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "facilitiesOrServiceNeededSubHeadingLabel",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```
Comment (akomisarek) [inline ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json:1090]: I don't think Court Admin should have perms to those links. Court admins can't trigger the events, can they?

File: ccd-definition/CaseTypeTab/CareSupervision/2_ViewApplication-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,628 @@
...
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "ViewApplicationTab",
+    "TabLabel": "View application",
+    "TabDisplayOrder": 2,
+    "CaseFieldID": "orders",
+    "TabShowCondition": "[STATE]=\"Open\" OR [STATE]=\"RETURNED\"",
```
Comment (akomisarek) [inline ccd-definition/CaseTypeTab/CareSupervision/2_ViewApplication-nonprod.json:42]: Any particular reason why this TabShow condition is not on the first element?

File: ccd-definition/CaseTypeTab/CareSupervision/8_Confidential.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "Channel": "CaseWorker",
     "TabID": "Confidential",
-    "TabLabel": "Confidential",
+    "TabLabel": "Confidential information",
```
Comment (akomisarek) [inline ccd-definition/CaseTypeTab/CareSupervision/8_Confidential.json:7]: is this expected change in scope of this story? I don't see anything in story/AC about it.

File: e2e/pages/caseView.page.js [hunk 4]
```diff
@@ -11,12 +11,13 @@ module.exports = {
     legalBasis: 'Legal basis',
     documents: 'Documents',
     documentsSentToParties: 'Documents sent to parties',
-    confidential: 'Confidential',
+    confidential: 'Confidential information',
```
Comment (akomisarek) [inline e2e/pages/caseView.page.js:14]: Same comment as for tab.

File: ccd-definition/CaseTypeTab/CareSupervision/8_Confidential.json (hunk 3 above)
Comment (Jamesnelson117) [reply ccd-definition/CaseTypeTab/CareSupervision/8_Confidential.json:7]: Suggest by design, comment is left on the ticket.

File: ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json (hunk 1 above)
Comment (Jamesnelson117) [reply ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json:1090]: Good point no let's remove.
